frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -76,7 +76,7 @@ const ODCEmptyState: React.FC<Props> = ({
           href={formatNamespacedRouteForResource('import', activeNamespace)}
           title="YAML"
           iconImg={yamlIcon}
-          description="Create or replace resources from their YAML or JSON definitions."
+          description="Create or replace resources from their YAML or JSON definitions"
```

Comment:
Replace is not supported, so we should update this one too

Reply:
When in Rome :)

Reply:
fyi @serenamarie125

Reply:
LGTM!